Cubical/Foundations/Prelude.agda

```diff
@@ -61,6 +61,15 @@ cong₂ : ∀ {C : (a : A) → (b : B a) → Type ℓ} →
         PathP (λ i → C (p i) (q i)) (f x u) (f y v)
 cong₂ f p q i = f (p i) (q i)
 
+-- The simplest, one-dimensional hcomp
+-- sometimes needed to overcome a termination error
+
+simpleHcomp : ∀ {w x y z : A} → w ≡ z → w ≡ x → x ≡ y → z ≡ y
```

Comment:
This is almost the same as `Cubical.Foundations.GroupoidLaws._∙∙_∙∙_` up to path directions, by the way.

Reply:
Writing down some notes:

https://github.com/agda/cubical/blob/7e6c71fa51b1633cd734e60eb8a6e7ba2707d4ea/Cubical/Foundations/GroupoidLaws.agda#L187-L188

Reply:
Yeah, I'm not sure we need this function as well then

Reply:
Fixed